### ISSUE TITLE
Only run update_biocontainers on containers in the registry

### ISSUE THREAD

**Issue by georgiastuart** [singularityhub/shpc-registry#103]
**Describe the bug**

Update biocontainers runs on a set of containers not necessarily in the registry. 

**To Reproduce**

- Create your own registry
- Delete all the biocontainers
- `update_biocontainers` still runs on all biocontainers

**Expected behavior**

I'd expect it to skip containers that aren't in the registry.

**Version of Singularity and Singularity Registry HPC Client**

https://github.com/georgiastuart/resistry-test

**Comment by georgiastuart** [comment]
I think I can fix this, I just can't do it right away so I wanted to bookmark with an issue!

**Comment by vsoch** [comment]
The listing comes from the galaxy project, here https://github.com/georgiastuart/resistry-test/blob/0a803915bcc1ae525e41248721c123eb8a869462/.github/scripts/get_biocontainers.py#L34 and the purpose is to add biocontainers that aren't in the registry. Can you help me understand why this needs fixing?

**Comment by vsoch** [comment]
Yes - we should definitely add a “delete this workflow file” to the developer tutorial! Do you want to add it or would you like me to?

**Comment by georgiastuart** [comment]
I can! I should have time later today.

**Comment by vsoch** [comment]
yay! Thank you! <3
